functions.py

```diff
@@ -461,5 +485,21 @@ def end_page():
         out_msg()
 
 
+def list_for_tomorrow():
+    foods_for_tomorrow = file_read_func()[2]
+    foods_for_tomorrow_string = ', '.join(foods_for_tomorrow)
+    if not foods_for_tomorrow:
+        msg = "За утре няма избрани ястия."
+    else:
+        msg = f"\nСписъкът за утре е:" \
+          f"\n{foods_for_tomorrow_string.upper()}"
+    options = ["НАЧАЛО", "ИЗХОД"]
+    next_step = buttonbox(msg=msg, title=version, choices=options)
+    if next_step == "НАЧАЛО":
+        home_page()
+    elif next_step == "ИЗХОД":
+        out_msg()
```

Comment:
Changing this to `else` would cover the case of pressing `X` for closing the message box and would be consistent with most of the other message boxes' cases too